src/Components/SmartComponents/CVEs/CVEsTableToolbar.test.js

```diff
@@ -23,6 +25,11 @@ jest.mock('../../../Store/Actions/Actions', () => ({
     }) }
 ));
 
+jest.mock('../../../Helpers/Hooks', () => ({
+    ...jest.requireActual('../../../Helpers/Hooks'),
+    useHybridSystemFilterFlag: jest.fn(() => true)
```

Comment:
Shouldn't this be default `false` as well and set to true only on line `232`?

Reply:
@bastilian Yup, I have made the change and also added test cases covering the flag is both off and on. PTAL

Reply:
@bastilian can I ask for your review on this? Thanks